### PR TITLE
libbitcoin-client 3.5.0 (new formula)

### PR DESCRIPTION
**Pull request by sugarjig**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

This is a new formula for [libbitcoin-client](https://github.com/libbitcoin/libbitcoin-client). This change also updates `libbitcoin-explorer` to use `libbitcoin-client` as a dependency instead of defining it as a resource.

### CODE REVIEW

File: Formula/libbitcoin-explorer.rb [hunk 1]
```diff
@@ -16,39 +16,15 @@ class LibbitcoinExplorer < Formula
...
-  end
-
-  resource "libbitcoin-client" do
-    url "https://github.com/libbitcoin/libbitcoin-client/archive/v3.5.0.tar.gz"
-    sha256 "bafa26647f334ecad04fc4bbef507a1954d7e0682f07bd38b90ab66dba5fe0d2"
-  end
-
   def install
     ENV.prepend_path "PKG_CONFIG_PATH", Formula["libbitcoin"].opt_libexec/"lib/pkgconfig"
+    ENV.prepend_path "PKG_CONFIG_PATH", Formula["libbitcoin-client"].opt_libexec/"lib/pkgconfig"
```
Comment (ilovezfs) [inline Formula/libbitcoin-explorer.rb]: This gets added automatically if it's a dependency so the line is not needed

Comment (sugarjig) [reply Formula/libbitcoin-explorer.rb]: I thought so too, but if I don't include it I get

```
Package 'libbitcoin-protocol', required by 'libbitcoin-client', not found
```

Comment (ilovezfs) [reply Formula/libbitcoin-explorer.rb]: Yes you're right. I didn't see the `libexec`

**Comment by ilovezfs** [comment]
thanks again @sugarjig

**Pull request closed without merging**